src/drafter.cc

```diff
@@ -0,0 +1,204 @@
+// vi:cin:et:sw=4 ts=4
+//
+//  drafter.cc
+//
+//  Created by Jiri Kratochvil on 2015-02-11
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+//
+//
```

Comment:
nitpick: These 2 extra empty comment lines are not needed. Just 1 empty comment line after the `Copyright` line is enough. Similarly in all the other files.